maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -3456,19 +3482,20 @@
 	dir = 8
 	},
 /obj/structure/surface/table/almayer,
-/obj/structure/machinery/computer/atmos_alert{
-	dir = 4;
-	pixel_y = 5
+/obj/item/paper{
+	desc = "Tackle is today. I am leaving this note so that the Chaplain who follows me may remember me. While the outlook is not good, I am sure that my faith will protect me as it will protect you. Signed <i>[the name has been blotted out by black ink]</i>";
```

Comment:
```suggestion
	desc = "Tackle is today. I am leaving this note so that the Chaplain who follows me may remember me. While the outlook is not good, I am sure that my faith will protect me as it will protect you. Signed <i>a name blotted out by ink.</i>";
```